feeds/dk.json

```diff
@@ -0,0 +1,18 @@
+@@ -0,0 +1,17 @@
```

Comment:
This is probably a leftover of some Git operation. It does not belong in the file, right?

Reply:
You are right. I'm sorry I didn't spot it.